interface/forms/soap/templates/general_new.php

```diff
@@ -68,8 +68,8 @@
 			</table>
 			<div>
 				<!-- Save/Cancel buttons -->
-				<input type="button" id="save" value="<?php echo xlta('Save'); ?>"> &nbsp;
-				<input type="button" id="dontsave" class="deleter" value="<?php echo xlta('Cancel'); ?>"> &nbsp;
+				<input type="button" id="save" value="<?php echo xla('Save'); ?>"> &nbsp;
+				<input type="button" id="dontsave" class="deleter" value="<?php echo xla('Cancel'); ?>"> &nbsp;
```

Comment:
Just for reference:
https://github.com/LibreHealthIO/LibreEHR/blob/master/library/htmlspecialchars.inc.php#L115